core/templates/pages/exploration-editor-page/preview-tab/preview-tab.component.spec.ts

```diff
@@ -229,50 +246,68 @@ describe('Preview Tab Component', function() {
       expect(ctrl.showParameterSummary()).toBe(true);
     });
 
-    it('should open set params modal', function() {
+    it('should open set params modal', fakeAsync(() => {
```

Comment:
Please add when this happens. Also, write the `it` block description from the user's perspective.

Reply:
Done.